surfaces_tools/widgets/pdos.py

```diff
@@ -40,13 +40,16 @@ def read_and_process_pdos_file(pdos_path):
     return out_data, kind
 
 
-def process_pdos_files(scf_calc, newversion=True):
+def process_pdos_files(pdos_wk, newversion=True):
```

Comment:
```suggestion
def process_pdos_files(pdos_workchain, newversion=True):
```
Please rename all the other occurrences as well.